src/pages/WorkloadDetails/WorkloadInfo/WorkloadPodLogs.tsx

```diff
@@ -182,6 +211,17 @@ export default class WorkloadPodLogs extends React.Component<WorkloadPodLogsProp
                         />
                       </ToolbarItem>
                     </ToolbarGroup>
+                    <ToolbarGroup>
+                      <ToolbarItem>
+                        <Switch
+                          id="simple-switch"
+                          label="Vertical"
+                          labelOff="Horizontal"
+                          isChecked={verticalOrientation}
```

Comment:
I actually find this confusing.  I don't think the label should toggle when the switch flips.  It's just "Vertical" on or "Vertical" off.   I think just remove the labelOff property.  Also, I think a label of "Side by Side" may be better than "Vertical" or "Horizontal" because it better indicates that we're talking about the window orientation.   For me I'd like the default to have the switch Off and the label be "Side by Side".  Regardless of what you choose for the label, the label should not toggle.

Reply:
@jshaughn that is a good point. I made those changes now.